cmd/ocm/list/machinepool/cmd.go

```diff
@@ -132,6 +133,13 @@ func printReplicas(autoscaling *cmv1.MachinePoolAutoscaling, replicas int) strin
 	return fmt.Sprintf("%d", replicas)
 }
 
+func printAdditionalSecurityGroups(securityGroups []string) string {
+	if len(securityGroups) == 0 {
+		return ""
+	}
```

Comment:
When the list is empty the join returns an empty string

Reply:
Shouldn't that be the expectation?

Reply:
ah sorry I did not explain myself properly, my point was that you don't need this part you can use just the join, you don't need this check

Reply:
Oh I see, interesting. There are lots of instances where we do this :laughing: